src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -135,9 +135,13 @@ Object.assign(SyntheticEvent.prototype, {
 
     if (event.stopPropagation) {
       event.stopPropagation();
-    } else {
+    } else if (typeof event.cancelBubble !== 'unknown') { // eslint-disable-line valid-typeof
+      // <= IE10 throws a "Member not found" error if the cancelBubble
+      // attribute is accessed from an onChange event. A typeof check of "unknown"
```

Comment:
Perhaps clarify that this is because of `"propertychange"` and not because of *actual* change events.